packages/graphql/src/schema/make-augmented-schema.ts

```diff
@@ -1031,9 +1035,33 @@ function makeAugmentedSchema(
                 const relatedNodes = nodes.filter((n) => connectionField.relationship.union?.nodes?.includes(n.name));
 
                 relatedNodes.forEach((n) => {
+                    const unionWhereName = `${connectionField.typeMeta.name}${n.name}Where`;
```

Comment:
Thanks for fixing the formatting on the TCK tests - allowed me to notice this.

Normally we would have like `MovieSearchGenreConnectionBlah`, but for this we have `MovieSearchConnectionGenreBlah` - like `MovieSearchConnectionGenreWhere`.

Really not a big deal and I can see how this was caused, and will approve anyway, just figured I would point it out in case you'd like to change it.

Reply:
I'm going to leave this, due to the fact we would have to split the `typeMeta.name` apart from the manually appended `connection` word and then rejoin it together - unnecessary logic IMO.